google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseTest.java

```diff
@@ -100,27 +102,13 @@ public void listDatabaseOperations() {
   @Test
   public void fromProto() {
     Database db = createDatabase();
-    assertThat(db.getId().getName()).isEqualTo(NAME);
-    assertThat(db.getState()).isEqualTo(DatabaseInfo.State.CREATING);
-    assertThat(db.getVersionRetentionPeriod()).isEqualTo(VERSION_RETENTION_PERIOD);
-    assertThat(db.getEarliestVersionTime()).isEqualTo(EARLIEST_VERSION_TIME);
-    assertThat(db.getEncryptionConfig())
-        .isEqualTo(EncryptionConfigs.customerManagedEncryption(KMS_KEY_NAME));
-  }
-
-  @Test
-  public void testFromProtoWithEncryptionConfig() {
```

Comment:
why do we remove this test?

Reply:
We did not remove it, just changed the `assertThat` to `assertEquals`.